frontend/packages/topology/src/types.ts

```diff
@@ -129,6 +129,9 @@ export interface GraphEntity<E extends Graph = Graph, D = any> extends ElementEn
   getEdges(): EdgeEntity[];
   getScale(): number;
   setScale(scale: number): void;
+  getLayout(): string;
```

Comment:
I think it should be possible to have no layout.
```suggestion
  getLayout(): string | undefined;
```